x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_search_bar/analytics_search_bar.tsx

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Dispatch, SetStateAction, FC, Fragment, useState } from 'react';
+import {
+  EuiSearchBar,
+  EuiSearchBarProps,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFormRow,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { stringMatch } from '../../../../../util/string_utils';
+import {
+  Clause,
+  TermClause,
+  FieldClause,
+  Value,
+  DataFrameAnalyticsListRow,
+} from '../analytics_list/common';
+
+export function filterAnalytics(
```

Comment:
it'd be better to move search server-side

Reply:
This search is just filtering the jobs that have already been loaded in response to what's being typed into the search bar. Could you explain a bit more about what you mean? I'm not sure how moving it to the server-side would benefit.

Reply:
Since you are replacing the in-memory table, it would be better to fetch only an active page instead of a complete list of jobs. Hence the kibana endpoint can receive `from`, `size` and `query` params to fetch the list from ES

Reply:
Oh you're talking about for the pagination itself. I'm not sure it's actually doing anything for us unless the user has a ton of jobs. This behavior is also consistent with anomaly detection jobs - which arguably users right now have many more of. I don't think this would be in the scope of this PR but if we do see any performance indicators once this is in, happy to change it.